rollup.config.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Again, it is better to change how we use `lint`, rather than disabling rules.

Reply:
Fixed 👍